File: Pratica 6 - Funcoes 2/cpp_logic/exercicio8_v1.cpp
#include <iostream>
using namespace std;

int main() {
    float lado_a, lado_b, lado_c;
    cout << "Digite o valor do lado A: ";
    cin >> lado_a;
    cout << "Digite o valor do lado B: ";
    cin >> lado_b;
    cout << "Digite o valor do lado C: ";
    cin >> lado_c;

    if (lado_a == lado_b && lado_b == lado_c) cout << "Equilátero";
    else if (lado_a == lado_b || lado_a == lado_c || lado_b == lado_c) cout << "Isósceles";
    else cout << "Escaleno";

    return 0;
}

File: Pratica 6 - Funcoes 2/python_logic/exercicio8_v1.py
lado_a = float(input("Digite o comprimento do lado a: "))
lado_b = float(input("Digite o comprimento do lado b: "))
lado_c = float(input("Digite o comprimento do lado c: "))

if lado_a == lado_b == lado_c:
    print("O triângulo é equilátero.")
elif lado_a == lado_b or lado_a == lado_c or lado_b == lado_c:
    print("O triângulo é isósceles.")
else:
    print("O triângulo é escaleno.")